cmd/nfd-worker/main.go

```diff
@@ -138,6 +136,8 @@ func initFlags(flagset *flag.FlagSet) (*worker.Args, *worker.ConfigOverrideArgs)
 		"Do not publish feature labels")
 	flagset.IntVar(&args.MetricsPort, "metrics", 8081,
 		"Port on which to expose metrics.")
+	flagset.IntVar(&args.GrpcHealthPort, "health", 8082,
```

Comment:
same conversation as in master

Reply:
I kept the helm value as `healthPort`, WDYT?